pushmi_pullyu.gemspec

```diff
@@ -19,7 +19,7 @@ Gem::Specification.new do |spec|
   spec.executables   = spec.files.grep(%r{^exe/}) { |f| File.basename(f) }
   spec.require_paths = ['lib']
 
-  spec.required_ruby_version = '>= 2.5'
+  spec.required_ruby_version = '>= 2.6'
```

Comment:
Again, should this be 2.7? When we went to 2.5 we had it at 2.5
 https://github.com/ualbertalib/pushmi_pullyu/commit/99b1d8f8882cdfcef153bbd19a5230b5ec969a4d

Reply:
We haven't built a pmpy to deploy with ruby 2.7 yet.  We haven't done that because there are some changes to pmpy on a server called chard that provide some alternate logging that haven't made it into this code base and in a rubygem release yet.  It would be least risky to have the option to build the next release against either 2.6 OR 2.7.

I probably made this change too big.  We noticed when we tried to merge the logging changes that the CI was complaining about a number of things.  This PR fixes that as well.  I can create a PR for just the CI fixes and then the Ruby version changes can go to 2.7 and be in a subsequent release rather than hedging or living in the past.

Reply:
That's ready to review: https://github.com/ualbertalib/pushmi_pullyu/pull/253